### DISCUSSION TITLE
Adding navigation

### DISCUSSION THREAD

**Discussion by Erallie** [ObsidianPublisher/obsidian-github-publisher#361] [Q&A — Plugin] [answered]
From my understanding, this plugin doesn't automatically insert any form of navigation into the site it hosts. Is this something I have to add in manually by inserting my own header in front of every note I publish? I've tried installing Jekyll templates to see if that fixes it, but none of the ones I try seem to work with the files I have in my repository.

I'm wanting to just include the files in a single folder within my already existing repository. I don't want to have to create a new one in order to try out Quartz to see if it solves my problem.

**Comment by ooker777** [comment, marked as answer]
Yes I'd say you need to do that yourself. This plugin only transform a markdown format that is convenient to you when using Obsidian to the markdown format that your SSG prefers. You have to config the SSG yourself.
